web/public/public.go

```diff
@@ -41,10 +42,30 @@ func Avatar(c echo.Context) error {
 	return echo.NewHTTPError(http.StatusNotFound, "Page not found")
 }
 
+// Prelogin returns information that could be useful to show a login page (like
+// in the flagship app).
+func Prelogin(c echo.Context) error {
+	inst := middlewares.GetInstance(c)
+	publicName, err := inst.PublicName()
+	if err != nil {
+		publicName = ""
```

Comment:
Is it possible to have no public name ? Should I set a default value in the login UI?

Reply:
Yes, it's possible (even if it should be really rare in practice).